src/runtime_src/doc/toc/xbtools_map.rst

```diff
@@ -18,152 +18,176 @@ You may refer the documentation of the new tools in the following links :ref:`xb
+|``validate``|``validate`` |``-d <bdf>``       |``--device <bdf>``    |
+|            |             |                   |                      |
++------------+-------------+-------------------+----------------------+
+|            |             |                   |                      |
+|            |             |``-q``             |``--run quick``       |
+|            |             |                   |                      |
++------------+-------------+-------------------+----------------------+
+|            |             |                   |                      |
+|            |             |      	       |``--run <testname>``  |
+|            |             |                   |                      |
++------------+-------------+-------------------+----------------------+
+|            |             |                   |                      |
+|            |             |      	       |``--format <type>``   |
+|            |             |                   |                      |
++------------+-------------+-------------------+----------------------+
+|            |             |                   |                      |
+|            |             |     	       |``--output <file>``   |
+|            |             |                   |                      |
++------------+-------------+-------------------+----------------------+
+|            |             |                   |                      |
+|``scan``    |``examine``  |                   |                      |
+|            |             |                   |                      |
++------------+-------------+-------------------+----------------------+
+|            |             |                   |                      |
+|``query``   |``examine``  |``-d <bdf>``       |``--device <bdf>``    |
+|            |             |                   |                      |
++------------+-------------+-------------------+----------------------+
+|            |             |                   |                      |
+|            |             |                   |``--report <list>``   |
+|            |             |                   |                      |
++------------+-------------+-------------------+----------------------+
+|            |             |                   |                      |
+|            |             |                   |``--output <file>``   |
+|            |             |                   |                      |
++------------+-------------+-------------------+----------------------+
+|            |             |                   |                      |
+|``dump``    |``examine``  |                   |``--format <type>``   |
+|            |             |                   |                      |
++------------+-------------+-------------------+----------------------+
+|            |             |                   |                      |
+|``top``     |``top``	   |``-d <bdf>``       |``--device <bdf>``    |
```

Comment:
We need to remove top as it will not work without the "--legacy" switch

Reply:
Removed top